server/vertx-server/src/test/scala/sttp/tapir/server/vertx/VertxServerTest.scala

```diff
@@ -2,11 +2,13 @@ package sttp.tapir.server.vertx
 
 import cats.effect.{IO, Resource}
 import io.vertx.core.Vertx
+import io.vertx.ext.web.{Route, Router}
 import sttp.monad.FutureMonad
 import sttp.tapir.server.tests._
+import sttp.tapir.server.vertx.streams.VertxStreams
 import sttp.tapir.tests.{Test, TestSuite}
 
-import scala.concurrent.ExecutionContext
+import scala.concurrent.{ExecutionContext, Future}
```

Comment:
remove this and .asInstanceOf - change `VertxTestServerInterpreter` to have `VertxStreams` instead of any. You will also have to change `route` contract in  VertxTestServerBlockingInterpreter.

Reply:
done